### DISCUSSION TITLE
User reset after lock

### DISCUSSION THREAD

**Discussion by MarcS1975** [Stirling-Tools/Stirling-PDF#897] [Q&A] [answered]
Hi there, first off thanks for an awesome product!
I had a user locked due to too many failed login attempts and would like to reset the user. I tried deleting and adding with same user name and newly assigned password but the user remains locked.
How can I reset a user?

**Comment by Frooodle** [comment, marked as answer]
> Hi there, first off thanks for an awesome product!
> I had a user locked due to too many failed login attempts and would like to reset the user. I tried deleting and adding with same user name and newly assigned password but the user remains locked.
> How can I reset a user?
> 

Restarting the application should do it

**Comment by MarcS1975** [comment]
restarting the container worked, thank you .
Is there a more elegant way to reset a user?

**Reply by Frooodle** [reply]
Not currently but it does reset next day
It's to avoid password cracking via spams etc

**Comment by Frooodle** [comment]
No not currently
